src/harness/testcases/WINNF_FT_S_FDB_testcase.py

```diff
@@ -519,45 +517,39 @@ def generate_FDB_3_default_config(self, filename):
 
     grant_g_b = json.load(
                 open(os.path.join('testcases', 'testdata', 'grant_0.json')))
-    # Set the grant frequency to overlap with the FSS 'E000306' which is 3625-3700 MHz.
-    grant_g_b['operationParam']['operationFrequencyRange'] = {
-        'lowFrequency': 3660000000,
-        'highFrequency': 3670000000
-    }
 
-    # Update the location 'X' of CBSD devices to be near FSS sites
-    # 'KA413' at Albright,WV
-    device_a['installationParam']['latitude'] = 39.57327
-    device_a['installationParam']['longitude'] = -79.61903
-
-    # 'E000306' at Andover,ME
-    device_b['installationParam']['latitude'] = 44.63367
-    device_b['installationParam']['longitude'] = -70.69758
-
-    # Creating conditionals for Cat B devices
-    self.assertEqual(device_b['cbsdCategory'], 'B')
-    conditionals_b = {
-        'cbsdCategory': device_b['cbsdCategory'],
-        'fccId': device_b['fccId'],
-        'cbsdSerialNumber': device_b['cbsdSerialNumber'],
-        'airInterface': device_b['airInterface'],
-        'installationParam': device_b['installationParam'],
-        'measCapability': device_b['measCapability']
+    device_a['installationParam']['latitude'] = 39.353414,
+    device_a['installationParam']['longitude'] = -100.195313
+
+    # Pre-load conditionals and remove reg conditional fields from registration
+    # request.
+    conditional_keys = [
+        'cbsdCategory', 'fccId', 'cbsdSerialNumber', 'airInterface',
+        'installationParam', 'measCapability'
+    ]
+    reg_conditional_keys = [
+        'cbsdCategory', 'airInterface', 'installationParam', 'measCapability'
+    ]
+    conditionals_a = {key: device_a[key] for key in conditional_keys}
+    device_a = {
+        key: device_a[key]
+        for key in device_a
+        if key not in reg_conditional_keys
     }
-    del device_b['cbsdCategory']
-    del device_b['airInterface']
-    del device_b['installationParam']
-    del device_b['measCapability']
 
-    conditionals = [conditionals_b]
+    fss_database_config = {
+        'hostName': 'localhost',
+        'port': 8003
+        'fileUrl': '/rest/fss/v1/allsitedata'
```

Comment:
Line   number 543 will prevent code from executing because ',' is missing at the end   of 'fileUrl': '/rest/fss/v1/allsitedata'

Reply:
Fixed.